packages/fast-components-react-msft/src/carousel/carousel.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Carousel",
+    "description": "A carousel component's schema definition.",
+    "type": "object",
+    "id": "carousel",
+    "properties": {
+        "label": {
+            "title": "Carousel accessible label",
+            "type": "string"
+        },
+        "activeId": {
+            "title": "Set active slide by HTML id attribute",
+            "type": "string"
+        },
+        "items": {
+            "title": "Slides",
+            "type": "array",
+            "items": {
+                "title": "Slide",
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "title": "HTML id attribute",
+                        "type": "string"
+                    },
+                    "theme": {
+                        "title": "Theme",
+                        "type": "string",
+                        "enum": [
+                            "dark",
+                            "light"
+                        ]
+                    },
+                    "content": {
+                        "title": "Slide content",
+                        "type": "object",
+                        "pluginId": "@microsoft/fast-components-react-msft/carousel/content"
```

Comment:
if this is meant to be a react child, it should not be in properties but in `reactProperties`